.github/workflows/release.yml

```diff
@@ -13,22 +11,23 @@ on:
 
 jobs:
   build-test-release:
+    # Run the job if the PR is from the same repository
```

Comment:
Just one comment here, how this github action is not fired from a PR should we do this check? And if so, should we do the same in qiskit.org? https://github.com/Qiskit/qiskit.org/blob/main/.github/workflows/deploy.yml#L14

Reply:
Because the `on` trigger is set for `workflow_dispatch`, `push` and `schedule`.
Not for `pull_request`.